spec/models/conference_spec.rb

```diff
@@ -778,6 +778,30 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+    let!(:conference) { create(:conference) }
+    let(:confirmed_unscheduled_event) { create(:event, program: conference.program, state: 'confirmed') }
+    let(:confirmed_scheduled_event) { create(:event, program: conference.program, state: 'confirmed') }
```

Comment:
you can create a `event_scheduled` instead of a normal event and then you don't need to create the event_schedule and to set the state. Look at: https://github.com/openSUSE/osem/blob/master/spec/factories/events.rb#L30